### ISSUE TITLE
Anti-saccharomyces cerevisiae antibodies 

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10890]
**Preferred term label:**
Anti-saccharomyces cerevisiae antibodies 

**Synonyms**
ASCA

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ulcerative colitis
Crohn disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001235
- label: Anti-Saccharomyces cerevisiae antibodies
- parents: [Bloodstream antibacterial antibody positivity (HP:0430061)](https://hpo.jax.org/app/browse/term/HP:0430061)
- pmid: PMID:3143445
- def: Antibodies against antigens presented by the cell wall of the yeast Saccharomyces cerevisiae.
- comment: Anti-Saccharomyces cerevisiae antibodies (ASCAs) have been used as a marker for Crohn disease, Behcet disease, celiac disease, and other conditions.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10890
- synonyms: ASCA(NONE)

**Comment by kanems** [comment]
@pnrobinson Shouldn't the parent for this new term instead be Bloodstream antifungal antibody positivity HP:0430076 https://hpo.jax.org/browse/term/HP:0430076 since S. cerevisiae is yeast and not bacteria? I don't know if perhaps this is a clinical categorization vs. a taxonomic distinction.
